### ISSUE TITLE
onclone event for drag and drop

### ISSUE THREAD

**Issue by Tyeboe** [CMoore97/JBEC#44]
@CMoore97 Could you add a method that I can call into passing a callback function accepting an element as a parameter that would allow me to receive all cloned elements when they are created?

You could just manage an array of function and execute them all when a clone occurs.

**Comment by Tyeboe** [comment]
@CMoore97  You there?

**Comment by CMoore455** [comment]
Yeah I'm here, I think I can do that.

**Comment by CMoore455** [comment]
@Tyeboe The way I have read your request, I'm understanding that you want the cloned element returned to you after it has been cloned? If that's the case then I made a function that has no need for a callback function, or the use for an array of functions. I have read that comment so many times and I still have no idea why I would need to use anything more that have a function that just takes in the element and returns it. I probably again have not fully understood what you wanted but I wrote what I wrote, so we'll look at it on Monday.
